src/components/dashboard/GovernanceSection/GovernanceSection.tsx

```diff
@@ -13,15 +13,22 @@ import { SafeAppsTag, SAFE_APPS_SUPPORT_CHAT_URL } from '@/config/constants'
 import { useDarkMode } from '@/hooks/useDarkMode'
 import { OpenInNew } from '@mui/icons-material'
 import NetworkError from '@/public/images/common/network-error.svg'
+import useChainId from '@/hooks/useChainId'
+import { getSafeTokenAddress } from '@/components/common/SafeTokenWidget'
 
 const GovernanceSection = () => {
+  const chainId = useChainId()
   const isDarkMode = useDarkMode()
   const theme = isDarkMode ? 'dark' : 'light'
   const { getAllowedFeaturesList } = useBrowserPermissions()
   const [claimingSafeApp, errorFetchingClaimingSafeApp] = useRemoteSafeApps(SafeAppsTag.SAFE_CLAIMING_APP)
   const claimingApp = claimingSafeApp?.[0]
   const fetchingSafeClaimingApp = !claimingApp && !errorFetchingClaimingSafeApp
 
+  if (!getSafeTokenAddress(chainId)) {
+    return null
+  }
```

Comment:
We could also create a wrapper component on top of GovernanceSection with this check to avoid calling all those hooks needlessly.

Reply:
Good point. I'll add this now.